### DISCUSSION TITLE
i got SSL error. im installed certificate and when i start my code on cmd nothing happens

### DISCUSSION THREAD

**Discussion by Avtandilov** [Rapptz/discord.py#5557] [Q&A]
![discord bnot](https://user-images.githubusercontent.com/45468906/90525243-b8711d00-e177-11ea-9219-fc4824719062.png)
![Screenshot_4](https://user-images.githubusercontent.com/45468906/90528445-4c90b380-e17b-11ea-9b51-2fc58ba6b2d1.png)

### Checklist

- [ ] I have searched the open issues for duplicates.
- [ ] I have shown the entire traceback, if possible.
- [ ] I have removed my token from display, if visible.

**Comment by numbermaniac** [comment]
You've accidentally clicked in the Command Prompt window so the display (and possibly the script) is paused. Press the Escape key on your keyboard to de-activate Select mode and resume execution.

Also note that your code will not print anything to the command prompt window when the bot connects - to test whether it's running, you'll want to try running the command that you made.

**Comment by Avtandilov** [comment]
> de-activate Select mode

i press escape key and nothing happens what to do?

**Comment by alkatar21** [comment]
Add the following to your code to activate a basic logging and than there should be output:
```py
import logging

logging.basicConfig(level=logging.INFO)
```
There is no logging by default. Since this is no library problem, if you need more help I'd suggest you join the discord server and ask in one of the the help channels.

**Comment by MuffinAmor** [comment]
Yo, SSL Certificate Error with Python beginns at the most with the installation on the System. 

From my site, i fail most when i execute the make command while the installation on a Debain10 System. 

My Advice is to try to reinstall Python on the System with root/admin permissions. 

Or try it to download it from the Mircosoft Store or use Anaconda for the  interpreter.

**Comment by crazygmr101** [comment]
Quoting Python Support server (this is pinned as this issue comes up a lot)

How to fix SSL Certificate issue on [Windows]

Sectigo (cert vendor) provides a download link of cert manually:
https://crt.sh/?id=2835394 <- gotten from the link below
You should find it at the bottom left corner, where its saying Download Certificate: PEM
If you can't find it: https://beans-took-my-kids.reeee.ee/38qB2n.png

You have to setup certificate yourself, you can just click on it, or if that doesn't work, refer here:
https://portal.threatpulse.com/docs/sol/Solutions/ManagePolicy/SSL/ssl_chrome_cert_ta.htm

How to fix SSL Certificate issue on [Mac]

Navigate to your Applications/Python 3.6/ folder and double click the Install Certificates.command to fix this.
That's it!
Happy Coding:owoYay:!

For those curious, it was caused by this:

https://support.sectigo.com/Com_KnowledgeDetailPage?Id=kA03l00000117LT
